src/compile_model_manifest.py

```diff
@@ -0,0 +1,30 @@
+import os
+import requests
+import yaml
+import re
+import json
+
+compiled_models = []
+preserved_keys = ["config_url", "applications", "download_url", "name", "description", "cite", "authors", "documentation", "tags", "covers"]
+assert 'url' not in preserved_keys
+with open('./src/models.yaml') as f:
+    model_list = yaml.safe_load(f.read())
+    for item in model_list:
+        config_url = item['config_url']
+        response = requests.get(config_url)
+        if response.status_code == 200:
```

Comment:
It's nice to use guard clause when possible to keep the indentation level low, especially for larger blocks.
```suggestion
        if response.status_code != 200:
            continue
```